pallets/roles/src/lib.rs

```diff
@@ -36,12 +36,16 @@ pub use types::*;
 #[cfg(test)]
 mod mock;
 
-#[cfg(test)]
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+#[cfg(all(test, not(feature = "runtime-benchmarks")))]
```

Comment:
because of this:
```rs
    #[cfg(feature = "runtime-benchmarks")]
    type SpacePermissionsProvider = Spaces;
    #[cfg(not(feature = "runtime-benchmarks"))]
    type SpacePermissionsProvider = Self;
 ```
 
 SpacePermissionsProvider is implemented only for mocks that provide stubs that cannot be used for benchmarking.
 
 but all tests rely on these stubs, so when benchmarking are enabled I don't run tests